### PR TITLE
fixed icons size and update copyright year

### PR DESCRIPTION
**Pull request by amanr-dev**

### CODE REVIEW

**Comment by amanr-dev** [comment]
For the example

Before :
Screenshot 2023-12-21 232352

After : 
Screenshot 2023-12-21 232415

**Comment by amanr-dev** [comment]
Hi @TorstenDittmann , please review the PR.

**Review by TorstenDittmann — APPROVED** [review]

**Review by TorstenDittmann — CHANGES_REQUESTED** [review]

File: packages/icons/scripts.js [hunk 1]
```diff
@@ -5,36 +5,36 @@ import { resolve } from "path";
...
-        useCSSVars: true
-    });
-}
+  await svgtofont({
+    src: src,
+    dist: resolve(process.cwd(), "dist"),
+    fontName: "icon",
+    styleTemplates: resolve(process.cwd(), "templates"),
+    css: {
+      fontSize: "22px",
```
[the remark this answers is not in the document]
Comment (amanr-dev) [reply packages/icons/scripts.js:14]: Ok, I will change this to the previous size.

[the remark this answers is not in the document]
Comment (amanr-dev) [reply packages/icons/scripts.js]: I am working on it

[the remark this answers is not in the document]
Comment (amanr-dev) [reply packages/icons/scripts.js]: Added font size to the icons individually.

**Comment by amanr-dev** [comment]
Hi @TorstenDittmann, now you can review the PR

**Pull request closed without merging**